after/syntax/yaml.vim

```diff
@@ -40,7 +40,7 @@ syn keyword yamlConstant NULL Null null NONE None none NIL Nil nil
 syn keyword yamlConstant TRUE True true YES Yes yes ON On on
 syn keyword yamlConstant FALSE False false NO No no OFF Off off
 
-syn match  yamlKey	"^\s*\zs[^ \t\"]\+\ze\s*:"
+syn match  yamlKey	"^\s*\zs[^ \t\"#]\+\ze\s*:"
```

Comment:
It looks as though this and the following line should match. While you're here, do you mind including a `\'` to treat strings with colons as strings? For example,

```yaml
'foo: baz'
```

```suggestion
syn match  yamlKey	"^\s*\zs[^ \t\"\'#]\+\ze\s*:"
```